### ISSUE TITLE
A11y_AzureToolsforVSCode_AzureFunction_Selectalocationfornewresources_ScreenReader: The NVDA announcing incorrect list markup for "Recommended" and "Other" list items, when control is in collapsed state.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/vscode-azurefunctions#2944]
### Actual:
When the "Recommended" and "Other" list items are in collapsed state, the NVDA announces incorrect list markup. For "Recommended" list item it announces "chevron-right Recommended 3 of 3" and for "chevron-right Other 2 of 3". But there are only 2 list is present when all controls are in collapsed state.

**This issue is also observed with JAWS.**

### Expected:
The NVDA should announce correct list markup. For "Recommended" list item it announces "Recommended 1 of 2 collapsed " and for "Other 2 of 2 collapsed"

**GitHubTags:**#A11yMAS;#A11yTCS;BM_AzureToolsforVSCode_Win32_Aug2021;#Azure Tools for VS
Code;#A11ySev2;#WCAG1.3.1;#DesktopApp;# Win10;#NVDA;#JAWS;#Element:Header;

### Environment Details:
Visual Studio Code
Azure Function
Operating System: Windows 10 Pro 21H1
Screen Reader: NVDA(2021.1),JAWS(2021.2107.12 ILM)

### Steps to Reproduce:
1. Launch the Visual studio code.
2. TAB to on "Extension" button and search for "Azure Functions" and select it using ENTER key.
3. "Azure Function" screen should open. TAB to "Install" button and hit ENTER key to install it.
4. "Azure" button will appear in navigation pane. TAB to it and select it.
5. Start NVDA.
6. TAB to any Subscription and expand options using Shift+F10 then TAB to "Create Function App in Azure" control and select it.
7. Edit some name for new function and hit ENTER key, then select "Powershell 7.0" as a "Runtime stack" and hit ENTER key.
8. "Select a location for new resources" edit control will appear. TAB to "Recommended" list item and collapsed it using ENTER key if the control is in expanded state and do same for the "Other" list item. 
9. Observe the announcement by screen reader for the "Recommended" list item.

### User Impact:
The screen reader users will get incorrect information about the number of list items.

“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by ejizba** [comment]
The existing grouping is a custom implementation for our extension. We want to use VS Code's grouping, which should be fully accessible, but unfortunately it's not exposed to extensions yet. Tracking issue here: https://github.com/microsoft/vscode/issues/74967

**Comment by 12-shweta** [comment]
We will track this bug using the provided tracking bug id https://github.com/microsoft/vscode/issues/74967

**Comment by MicroFish91** [comment]
Fixed by https://github.com/microsoft/vscode-azuretools/pull/1165

**Comment by jinglouMSFT** [comment]
@12-shweta The work has been merged into the vscode-azuretools repo. Can you try to install the latest vsix to verify the fix?  Please address any questions to MicroFish91 as ejizba is no longer on this project anymore.  Thanks.

**Comment by 12-shweta** [comment]
Verified the bug and the issue is fixed.
Hence closing the bug.
![image](https://user-images.githubusercontent.com/54656883/172593771-f1fcfb2d-6d9d-4685-a654-7fa2e08cb832.png)

**Comment by Rama8345** [comment]
**Github Tags:**#Closed;

**Comment by InduPriya1805** [comment]
GitHubTags:#BM_AzureToolsforVSCode_Win32_Aug2021;
